src/webhook.rs

```diff
@@ -40,8 +40,11 @@ pub async fn webhook(bot: Bot) -> mpsc::UnboundedReceiver<Result<Update, String>
 
     let teloxide_token = env::var("TELOXIDE_TOKEN").expect("TELOXIDE_TOKEN env variable missing");
     let host = env::var("HOST").expect("HOST env variable missing");
-    let path = format!("/{}/api/v1/message", teloxide_token);
-    let url = format!("https://{}{}", host, path);
+    let path = match env::var("PATH"){
```

Comment:
Also I can suggest to merge `HOST` and `WEBHOOK_URI` into the one env var. Then the url will be like `let url = path;`